modules/imgimport/CMakeLists.txt

```diff
@@ -1,2 +1,8 @@
-include_directories(include ../core/include)
-add_library(ImageImport src/imgimport.cpp)
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+include_directories(include ../core/include ${DeckLinkSDK_INCLUDE_PATH})
+ADD_DEFINITIONS("-DBOOST_LOG_DYN_LINK")
+add_library(ImageImport ${BOOST_LIBRARIES} ${DeckLinkSDK_LIBS}  src/imgimport.cpp src/DeckLinkCapture.cpp src/DeckLinkOpenCv.cpp src/DeckLinkInputCallback.cpp)
```

Comment:
Does this actually work? I wasn't aware that you could link libraries in add_library, though maybe this is why you linked Decklink into the main executable?